index.md

```diff
@@ -5,8 +5,11 @@ permalink: /
 ---
 
 
-**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents.
+**JSON Schema** enables the confident and reliable use of the JSON data format.
```

Comment:
I think the original language here was a bit more concrete (i.e. corresponded directly to specific functionality). So maybe we leave both? E.g.

```suggestion
**JSON Schema** is a vocabulary that allows you to **annotate** and **validate** JSON documents. It enables the confident and reliable use of the JSON data format.
```